soundfile.py

```diff
@@ -12,17 +12,39 @@
 
 import os as _os
 import sys as _sys
+import numpy as np
```

Comment:
I prefer to just `import numpy`, without renaming it to `np`.

In this file, we're importing all names with leading underscores, to keep a clean module namespace. This was not my brightest ideas, all things considered. But now that we started it, we should keep it like that.

Reply:
Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/a8a77c1e8155dc029f53aad1d69a7a501a60a671